core/code/utils_misc.js

```diff
@@ -483,7 +491,7 @@ window.makePermalink = function (latlng, options) {
   options = options || {};
 
   function round (l) { // ensures that lat,lng are with same precision as in stock intel permalinks
-    return Math.trunc(l*1e6)/1e6;
+    return Math.floor(l*1e6)/1e6;
```

Comment:
`Math.trunc` and `Math.floor` have different values on negative numbers. Maybe we should continue to use `Math.trunc`, since a polyfill has been added?

Reply:
> `Math.trunc` and `Math.floor` have different values on negative numbers.

Right. But in fact `Math.trunc` choice was arbitrary from the beginning. E.g. we could use `Math.round` instead, with similar consequences (0.5:1, -0.5:0).
Anyway I believe that here it does not matter at all, as for permalink there is no need in precise calculations (in any sense).

I can revert to `Math.trunc` if you still feel that more correct.